### PR TITLE
Enable dashboards in Zenkoversion

### PR DESCRIPTION
**Pull request by francoisferrand**
Issue: ZENKO-3843

### CODE REVIEW

**Review by lavigne958 — APPROVED** [review]

**Review by lavigne958 — APPROVED** [review]

File: solution/build.sh [hunk 1]
```diff
@@ -175,7 +177,7 @@ EOF
 
 function get_local_dashboards()
 {
-    find ${REPOSITORY_DIR}/monitoring/ -type d -depth 1 -print0 |
+    find ${REPOSITORY_DIR}/monitoring/ -maxdepth 1 -type d -print0 |
```
Comment (lavigne958) [inline solution/build.sh]: what a fantastic fix ! :heart:

File: solution/build.sh [hunk 2]
```diff
@@ -87,7 +87,9 @@ function copy_yamls()
     mkdir -p ${crd_dir} ${role_dir}
 
     kustomize build "${zenko_operator_repo}/config/artesca-solution/crd?ref=$(zenko_operator_tag)" -o ${crd_dir}
-    rename '.yaml' '_crd.yaml' ${crd_dir}/*
+    for file in ${crd_dir}/*.yaml ; do 
+        mv $file ${file%.yaml}_crd.yaml
+    done
```
Comment (lavigne958) [inline solution/build.sh:92]: for those who don't have `rename`:  https://github.com/util-linux/util-linux

**Review by lavigne958 — APPROVED** [review]

File: solution/build.sh (hunk 2 above)
Comment (francoisferrand) [reply solution/build.sh:92]: worth noting the binary is actually called `rename.ul` in ubuntu, though..

**Review by lavigne958 — APPROVED** [review]

**Review by XinLi0207 — APPROVED** [review]

**Review by lavigne958 — APPROVED** [review]

**Pull request merged**